src/jit/InstList.cpp

```diff
@@ -255,7 +255,7 @@ BrTableInstruction* JITCompiler::appendBrTable(ByteCode* byteCode, uint32_t numT
 
 void JITCompiler::dump()
 {
-    bool enableColors = (verboseLevel() >= 2);
+    bool enableColors = (JITFlags() & JITFlagValue::JITVerboseColor);
```

Comment:
This `enableColors` is defined here but seems never used afterward.

Reply:
The dump sets coloring escape sequences based on it:
https://github.com/Samsung/walrus/blob/main/src/jit/InstList.cpp#L267